### DISCUSSION TITLE
What are the different output options?  (and doublescan option?)

### DISCUSSION THREAD

**Discussion by sofakng** [joncampbell123/dosbox-x#4335] [General]
Is there a guide or detailed explanation of the different output options?  (ie. surface, opengl, openglnb, openglpp, etc)

Also, can somebody explain the doublescan option?

Thanks!

**Comment by Torinde** [comment]
the dosbox-x.reference.conf file has descriptions of most options:
![image](https://github.com/joncampbell123/dosbox-x/assets/97228894/59b1e2d2-4c50-4613-b507-00431d3d00d1)
Also look at: http://dosbox-x.com/wiki/

**Reply by sofakng** [reply]
Thanks for the information.  I'm trying to read as much as possible but the render pipeline still seems complex and confusing.

For example, is the scaler setting still recommended?  DOSBox Staging says it deprecated and really shouldn't be used.  Is this the same for DOSBox-X?

**Comment by rderooy** [comment]
These render pipeline improvements that Staging made have (until now) not been ported to DOSBox-X. At this point the codebase of Staging and X has diverged quite a bit, making porting of code, especially large features quite involved.

**Comment by RBTCHENTPE** [comment]
Dear Joncampbell,

 

See & understand,

Thanks a lot for your kind reply,

 

By the way, would you please kindly advise me the right application of Edit or Qbasic for editing the text files as well as the old dos commands.

 

I found the Edit & Qbasic is not able to access the files of text in DOSBOX-X.

 

B. RGDS.,

ROBERT CHEN

**Comment by RBTCHENTPE** [comment]
Dear Joncampbell,

 

Refer to the attachment, dos command PRINT is not workable.

 

The dir > prn is workable in good order.

 

Would you please kindly advise me the right dos command with syntax to print out the text files.

 

B. RGDS.,

ROBERT CHEN

 

DOSBOX-X DOS622 DOS COMMAND PRINT FILES FAIL

OS: WINDOWS 10 INSTALL NETWORK PRINTER SERVER OF ZERO ONE PA301, the Printfil & DOSPRINTER is installed and put the DOSPrinter.exe in C:\ DOSBox-X

IP: 192.168.1.10 SET LP1 = ANDY1, LP2 = ANDY2 LP3 = ANDY3

PRINTER : IS DOT MATRIX PRINTER

THE DOSBOX-X CONFIG PRINTER AS FOLLOWS:



1. dir *.bat > \\op4\andy1   ## sent dir *.bat to printer andy1 successful

Following command print files always fail:

2. dir *.bat > \\192.168.1.10\andy1   ## sent dir *.bat to printer andy1 fail

                alert: file creation error - \\192.168.1.10\andy1

3. print /d:\\op4\andy1 d:\988920.atc  ## fail

                alert: List output is not assigned to a device

4. print LPT1 /d:\\op4\andy1 d:\988920.atc  ## fail

                alert: NAME OF LIST DEVICE [prn]: ANDY1

                alert: List output is not assigned to a device

5. print LPT1: /d:\\op4\andy1 d:\988920.atc  ## fail

                alert: NAME OF LIST DEVICE [prn]: ANDY1

                alert: List output is not assigned to a device

描述: 描述: C:\DOSBox-X\capture\command_000.png

 

描述: 描述: C:\DOSBox-X\capture\command_001.png

 

 

 

 

 



-- 
這封電子郵件已經過 AVG 防毒軟體檢查病毒。
www.avg.com

**Comment by RBTCHENTPE** [comment]
Dear Joncampbell,

 

Refer to the attachment, dos command PRINT is not workable.

 

1.      The dir > prn is workable in good order.

 

2.      Would you please kindly advise me the right dos command with syntax to print out the text files.

 

3.      Would you please kindly advise me how to active the dos command of ping.exe for tracing network.

 

Your kind assistance with prompt reply will be highly appreciated

 

B. RGDS.,

ROBERT CHEN

 

DOSBOX-X DOS622 DOS COMMAND PRINT FILES FAIL

OS: WINDOWS 10 INSTALL NETWORK PRINTER SERVER OF ZERO ONE PA301, the Printfil & DOSPRINTER is installed and put the DOSPrinter.exe in C:\ DOSBox-X

IP: 192.168.1.10 SET LP1 = ANDY1, LP2 = ANDY2 LP3 = ANDY3

PRINTER : IS DOT MATRIX PRINTER

 

 

THE DOSBOX-X CONFIG NETWORK AS FOLLOWS:



 

THE DOSBOX-X CONFIG PRINTER AS FOLLOWS:

描述: ***@***.***

1. dir *.bat > \\op4\andy1   ## sent dir *.bat to printer andy1 successful

Following command print files always fail:

2. dir *.bat > \\192.168.1.10\andy1   ## sent dir *.bat to printer andy1 fail

                alert: file creation error - \\192.168.1.10\andy1

3. print /d:\\op4\andy1 d:\988920.atc  ## fail

                alert: List output is not assigned to a device

4. print LPT1 /d:\\op4\andy1 d:\988920.atc  ## fail

                alert: NAME OF LIST DEVICE [prn]: ANDY1

                alert: List output is not assigned to a device

5. print LPT1: /d:\\op4\andy1 d:\988920.atc  ## fail

                alert: NAME OF LIST DEVICE [prn]: ANDY1

                alert: List output is not assigned to a device

描述: 描述: 描述: C:\DOSBox-X\capture\command_000.png

 

描述: 描述: 描述: C:\DOSBox-X\capture\command_001.png

 

 

 

 

 



-- 
這封電子郵件已經過 AVG 防毒軟體檢查病毒。
www.avg.com
